metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -407,10 +414,20 @@ final class Doctor(
     }
   }
 
+  private def extractCompilationStatus(
+      targetId: BuildTargetIdentifier
+  ): DoctorStatus = {
+    val errors = diagnostics.getCompilationErrors(targetId)
```

Comment:
Did you check what happens (if any errors are shown) in case the compilation broke? Like with the release flag?

Reply:
now doctor should show `❌` for targets where compilation was aborted or ended with error status code

Reply:
Awesome, great! Now the doctor will finally not ignore people's issues :S